src/DataViews/DataViewTestUtils.h

```diff
@@ -24,7 +24,8 @@ void CheckCopySelectionIsInvoked(const std::vector<std::string>& context_menu,
 
 void CheckExportToCsvIsInvoked(const std::vector<std::string>& context_menu,
                                const MockAppInterface& app, DataView& view,
-                               const std::string& expected_contents);
+                               const std::string& expected_contents,
+                               bool is_exporting_events = false);
```

Comment:
How about instead of is_exporting_events, have this:
```
std::string context_menu_entry = "Export to CSV"
```
Then you can get rid of the line: `std::string action = is_exporting_events ? "Export events to CSV" : "Export to CSV";`

Reply:
Done! Thanks!